From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 23 Sep 2021 17:53:06 -0700
Subject: [PATCH] Unpin prettytable dependency

The version pin might not be necessary anymore. Preemptively remove the
pin for our package. I asked for clarification upstream:

https://github.com/catherinedevlin/ipython-sql/issues/202

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e4a653d..d831ee3 100644
--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,7 @@ NEWS = open(os.path.join(here, 'NEWS.rst'), encoding='utf-8').read()
 version = '0.4.0'
 
 install_requires = [
-    'prettytable<1',
+    'prettytable',
     'ipython>=1.0',
     'sqlalchemy>=0.6.7',
     'sqlparse',
